robot/Cumulus/tests/browser/zpayment_allocations/default_allocations_change_behavior.robot

```diff
@@ -0,0 +1,69 @@
+*** Settings ***  
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/OpportunityPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
+Suite Teardown  Run keywords
+...             Disable Default And Payment Allocations
+...             Delete Records and Close Browser
+
+*** Variables ***
+&{contact1_fields}         Email=test@example.com
+&{contact2_fields}         Email=test@example.com
+&{opportunity1_fields}     Type=Donation   Name=$0 opp with default allocations enabled     Amount=0    StageName=Prospecting   npe01__Do_Not_Automatically_Create_Payment__c=false   
+&{opportunity2_fields}     Type=Donation   Name=$0 opp with default allocations disabled    Amount=0    StageName=Prospecting   npe01__Do_Not_Automatically_Create_Payment__c=false 
+
+*** Test Cases ***
+Allocations Behavior when $0 with Default Allocations Enabled
+    [Documentation]             Enable payment allocation and make sure default allocations are enabled
+    ...                         Create a $0 opportunity.Default GAU allocation should still be created for opportunity. 
+    [tags]                      unstabe     W-035595    feature:Payment Allocations
+    Enable Default Allocations
+    Setupdata                   contact1            ${contact1_fields}     ${opportunity1_fields}
```

Comment:
Why can't the setup be moved to the global level

Reply:
setup has to be done after settings